server/koa.js

```diff
@@ -9,6 +9,7 @@ import mount from 'koa-mount';
 import helmet from 'koa-helmet';
 import logger from 'koa-logger';
 import favicon from 'koa-favicon';
+import proxy from 'koa-proxy';
```

Comment:
Move the require module into the condition, it will save time when starting server on production env.